### PR TITLE
MGMT-11124: support IMAGE_SERVICE_BASE_URL env var

### PR DESCRIPTION
**Pull request by danielerez**
## Description
As a preparation to remove the support for IMAGE_SERVICE_SCHEME:
* Added support for IMAGE_SERVICE_BASE_URL when should be used by all deployments instead of IMAGE_SERVICE_SCHEME/HOST.
* When IMAGE_SERVICE_BASE_URL is specified, it should be used instead of IMAGE_SERVICE_HOST/SCHEME.
* The current IMAGE_SERVICE_HOST/SCHEME vars are deprecated for now, until IMAGE_SERVICE_BASE_URL is exclusively used.

## How was this code tested?
Updated tests and verified on local env.

## Assignees
/cc @carbonin 

## Checklist

- [x] Title and description added to both, commit and PR
- [x] Relevant issues have been associated
- [x] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit tests (note that code changes require unit tests)

### CODE REVIEW

**Comment by carbonin** [comment]
Should this be WIP until https://github.com/openshift/assisted-image-service/pull/88 is merged since this includes both commits?

/hold

**Review by carbonin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ skipper make test
 - `ASSISTED_SERVICE_SCHEME` - protocol to use to query assisted service for image information
 - `ASSISTED_SERVICE_HOST` - host or host:port to use to query assisted service for image information
 - `IMAGE_SERVICE_SCHEME` - protocol to use to query the image service
```
[the remark this answers is not in the document]
Comment (danielerez) [reply README.md]: Yeah, already updated it in the latest version.

[the remark this answers is not in the document]
Comment (carbonin) [reply README.md]: Yeah I saw that but forgot this comment was sitting here from when I first looked at the PR.

**Pull request merged**